cluster/src/main/java/com/hartwig/pipeline/execution/vm/storage/LocalSsdStorageStrategy.java

```diff
@@ -0,0 +1,20 @@
+package com.hartwig.pipeline.execution.vm.storage;
+
+import static java.lang.String.format;
+import static java.util.Arrays.asList;
+
+import java.util.List;
+
+public class LocalSsdStorageStrategy implements StorageStrategy {
+    private static final int SSD_DEVICE_COUNT = 4;
```

Comment:
4 again? any relationship to the other constant?

Reply:
Yeah see the conversation up top. I realised this after I pushed.

Essentially in old-school hardware terms we're connecting four disks before we boot the machine, then we're formatting them once it's booted.

I'll pull them together.